backend/dataall/modules/dataset_sharing/api/resolvers.py

```diff
@@ -90,21 +90,25 @@ def resolve_user_role(context: Context, source: ShareObject, **kwargs):
         return None
     with context.engine.scoped_session() as session:
         dataset: Dataset = DatasetRepository.get_dataset_by_uri(session, source.datasetUri)
-        if (
-            dataset and (
+
+        can_approve = True if (
+                dataset and (
                 dataset.stewards in context.groups
                 or dataset.SamlAdminGroupName in context.groups
                 or dataset.owner == context.username
-            )
-        ):
-            return ShareObjectPermission.Approvers.value
-        if (
+        )
```

Comment:
there is some formatting issues with indentations on this piece that running the `make lint` command is complaining about - to resolve can have it as

```
        can_approve = True if (
            dataset and (
                dataset.stewards in context.groups
                or dataset.SamlAdminGroupName in context.groups
                or dataset.owner == context.username
            )
        ) else False

        can_request = True if (
            source.owner == context.username
            or source.groupUri in context.groups
        ) else False
```